usr.sbin/newsyslog/newsyslog.conf.5

```diff
@@ -117,7 +117,7 @@ This does not consider the current log file.
 .It Ar size
 When the size of the log file reaches
 .Ar size
-in kilobytes, the log file will be trimmed as described above.
+(Default in Kilobytes, you can specify K, M, G, ...), the log file will be trimmed as described above.
```

Comment:
I would prefer something along the lines:
```mdoc
in kilobytes, or by using any suffix (e.g., k, M, G) supported by
.Xr expand_number 3 ,
the log file will be trimmed as described above.
```
And a cross reference to this manual in the `SEE ALSO` section.

Reply:
![image](https://github.com/user-attachments/assets/a6f1dfcc-161a-48b3-b922-6a99c0c365f4)
![image](https://github.com/user-attachments/assets/c6a45141-0407-45f8-8418-211b202d29f4)

Reply:
Sorry to insist, but `kilobyte` should be written as is.
From https://en.wikipedia.org/wiki/Kilobyte:
> The internationally recommended unit symbol for the kilobyte is **kB**.[^1]

[^1]: https://en.wikipedia.org/wiki/Kilobyte#cite_note-IEC80000-1

Reply:
Sorry for my bad english.
Do you mean that I should write 'kilobyte' instead of 'kilobytes'?

Reply:
> Sorry for my bad english. Do you mean that I should write 'kilobyte' instead of 'kilobytes'?

With the letter "k" (lowercase).